<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LCCU5LL" failures="1" tests="1" name="Test3" time="35.006" errors="0" timestamp="2021-04-28T14:16:36 MDT">
  <testcase classname="tests.threeTest" name="threetest" time="27.12">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.threeTest.threetest(threeTest.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- threetest -->
</testsuite> <!-- Test3 -->
